website/docs/jp/switch.md

```diff
@@ -143,7 +143,7 @@
 
 | Attribute      | Description          | Type      | Accepted Values       | Default |
 |-----| ----| ----| ----|---- |
-| value / v-model | バインディング値 | boolean / string / number | — | — |
+| value / v-model | バインド値は、"active-value "または "inactive-value "と等しくなければなりません。デフォルトの型は "boolean "です。 | boolean / string / number | — | — |
```

Comment:
According to the context `バインディング` is the katakana expression of `binding` which can be written as `bindingu` in roman. So this value should stay the same as before. Also I think, not translating them is better than translating them without knowing the correctness of the translation, I'd like to ask you to change them into English instead of using translator which might cause misunderstanding to native speakers, unless you speaks these language. I as a reviewer only speaks Chinese/English/limited Japanese. So I can only review them.

Reply:
I understand. Because I'm not good for other languages, so can I only change Chinese documents and pull request?
I didn't submit it before, so I subconsciously thought I needed to make sure all the documents were consistent.